.gitignore

```diff
@@ -140,4 +140,5 @@ dmypy.json
 .vscode/
 
 #DS_Store
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
Maybe we need something like */.DS_Store to fix it permanently?

Reply:
There's no DS_Store files on my side though, or am I missing something?